packages/cli/src/commands/generate/service/service.js

```diff
@@ -148,7 +148,7 @@ export const buildStringifiedScenario = async (model) => {
       : typeof value === 'string' && value.match(/^\d+n$/)
       ? Number(value.substr(0, value.length - 1))
       : value
-  )
+  ).replace(/"(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z)"/g, 'new Date("$1")')
```

Comment:
I would take this opportunity to split the stringify callback function into separate if statements to make it more readable, and then add the date check in there as well. Plus add a comment about what/why we're doing. With long regexes like this one I also like to provide some example input so the reader knows what kind of strings the regex is supposed to work on.

What do you think about all that?

Reply:
yes, thanks for the feedback. the new PR will take care of it.